tasks/src/test/java/com/example/task/CreateQueueIT.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.example.task;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertNotNull;
+
+import com.google.cloud.tasks.v2.CloudTasksClient;
+import com.google.cloud.tasks.v2.QueueName;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for creating Tasks with HTTP targets. */
+@RunWith(JUnit4.class)
+public class CreateQueueIT {
+  private static final String PROJECT_ID = System.getProperty("GOOGLE_CLOUD_PROJECT");
+  private static final String LOCATION_ID = "us-central1";
+  private static final String QUEUE_ID = "test-queue-" + UUID.randomUUID();
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  private static void requireEnvVar(String varName) {
+    assertNotNull(
+        String.format("Environment variable '%s' must be set to perform these tests.", varName),
+        System.getProperty(varName));
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_CLOUD_PROJECT");
+  }
```

Comment:
Is there a reason for doing this rather than using the sample project like the create task test does?

Reply:
It makes it easier to run the tests in your own project without having to modify the project id in the code. It seems that tests added to the java-docs recently have been using this approach (ex. https://github.com/GoogleCloudPlatform/java-docs-samples/blob/c3bccc301b14b01924ed05f422160190dcd94646/dataproc/src/test/java/InstantiateInlineWorkflowTemplateTest.java) so I just followed that since that's how Python did it as well.

Reply:
Sounds good, thanks for the explanation. 

If you want you can update CreateTask tests while you're here to keep consistency within the /tasks directory.